core/utils.go

```diff
@@ -1 +1,80 @@
 package core
+
+import (
+	"fmt"
+	"github.com/refractionPOINT/go-limacharlie/limacharlie"
+	"path"
+	"strings"
+	"sync"
+)
+
+var (
+	secretCache = make(map[string]string)
+	cacheMutex  sync.RWMutex
+)
+
+func usingSecretValue(key string, org *limacharlie.Organization, fn func(val string) error) error {
```

Comment:
```suggestion
func UsingSecretValue(key string, org *limacharlie.Organization, fn func(val string) error) error {
```

needs to be public :)

Reply:
Beat me to it, I had just pushed that change